invenio_rdm_records/services/schemas/community_records.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Record communities schema."""
+
+from marshmallow import EXCLUDE, Schema, fields
+
+
+class RecordCommunitiesSchema(Schema):
+    """Record communities schema."""
+
+    communities = fields.List(fields.String())
```

Comment:
minor: I'm slightly in favour of a list of objects, for future-proofing and consistency e.g. with [the files init call](https://inveniordm.docs.cern.ch/reference/rest_api_drafts_records/#start-draft-file-uploads). So something like:

```python
{
  "communities": [
    {"id": "<community1_id>"},
    {"id": "<community2_id>"},
  ]
}
```